app/app.go

```diff
@@ -292,54 +292,105 @@ func (c colimaApp) SSH(args ...string) error {
 	return guest.SSH(workDir, args...)
 }
 
-func (c colimaApp) Status(extended bool) error {
+type status struct {
+	DisplayName  string `json:"display_name"`
+	Driver       string `json:"driver"`
+	Arch         string `json:"arch"`
+	Runtime      string `json:"runtime"`
+	MountType    string `json:"mount_type"`
+	IPAddress    string `json:"ip_address"`
+	DockerSocket string `json:"docker_socket"`
+	Kubernetes   bool   `json:"kubernetes"`
+	CPU          int    `json:"cpu"`
+	Memory       int64  `json:"memory"`
+	Disk         int64  `json:"disk"`
+}
+
+func (c colimaApp) getStatus() (*status, error) {
 	ctx := context.Background()
 	if !c.guest.Running(ctx) {
-		return fmt.Errorf("%s is not running", config.CurrentProfile().DisplayName)
+		return nil, fmt.Errorf("%s is not running", config.CurrentProfile().DisplayName)
 	}
 
 	currentRuntime, err := c.currentRuntime(ctx)
 	if err != nil {
-		return err
+		return nil, err
 	}
 
-	driver := "QEMU"
+	var status status
+	status.DisplayName = config.CurrentProfile().DisplayName
+	status.Driver = "QEMU"
 	conf, _ := configmanager.LoadInstance()
 	if !conf.Empty() {
-		driver = conf.DriverLabel()
+		status.Driver = conf.DriverLabel()
 	}
-
-	log.Println(config.CurrentProfile().DisplayName, "is running using", driver)
-	log.Println("arch:", c.guest.Arch())
-	log.Println("runtime:", currentRuntime)
-	if conf.MountType != "" {
-		log.Println("mountType:", conf.MountType)
+	status.Arch = fmt.Sprintf("%v", c.guest.Arch())
+	status.Runtime = currentRuntime
+	status.MountType = conf.MountType
+	ipAddress := limautil.IPAddress(config.CurrentProfile().ID)
+	if ipAddress != "127.0.0.1" {
+		status.IPAddress = limautil.IPAddress(config.CurrentProfile().ID)
```

Comment:
There is no need to do the lookup again.

```suggestion
		status.IPAddress = ipAddress
```